vendor/engines/secure_rooms/app/views/secure_rooms/occupancies/index.html.haml

```diff
@@ -22,3 +22,21 @@
           %td= occupancy.entry_at
 - else
   %p.notice= text("views.admin.secure_rooms.occupancies.index.none")
+
+- if @problem_occupancies.present?
+  %p= text("views.admin.secure_rooms.occupancies.index.problem")
+
+  %table.table.table-striped.table-hover.product_list
+    %thead
+      %tr
+        %th= User.human_attribute_name(:username)
+        %th= SecureRooms::Occupancy.human_attribute_name(:entry_at)
+        %th= SecureRooms::Occupancy.human_attribute_name(:exit_at)
+        %th= SecureRooms::Occupancy.human_attribute_name(:orphaned_at)
+    %tbody
+      - @problem_occupancies.each do |occupancy|
+        %tr.secure_rooms_problem_occupancy{id: "secure_rooms_problem_occupancy_#{occupancy.id}"}
+          %td= occupancy.user.username
+          %td= occupancy.entry_at
```

Comment:
How does this get formatted? I think we often use something like `human_datetime()` (a custom method of ours that's nil-safe).